geomesa-gt/geomesa-gt-spark/src/test/scala/org/locationtech/geomesa/geotools/spark/GeoToolsSpatialRDDProviderTest.scala

```diff
@@ -6,19 +6,16 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.locationtech.geomesa.spark.geotools
+package org.locationtech.geomesa.geotools.spark
```

Comment:
Hmm...  Have we settled on a scheme here?

I see
package org.locationtech.geomesa.accumulo.spark
package org.locationtech.geomesa.bigtable.spark
package org.locationtech.geomesa.bigtable.spark;
...
package org.locationtech.geomesa.fs.spark
package org.locationtech.geomesa.hbase.spark
package org.locationtech.geomesa.kudu.spark
package org.locationtech.geomesa.spark
package org.locationtech.geomesa.spark.accumulo
package org.locationtech.geomesa.spark.api.java
package org.locationtech.geomesa.spark.api.python
package org.locationtech.geomesa.spark.converter
package org.locationtech.geomesa.spark.geotools
package org.locationtech.geomesa.spark.hbase

Is this a preference for o.l.g.THING.spark?

Reply:
I was mainly thinking about package isolation for java 11. If we want to be able to put anything into `org.locationtech.geomesa.spark` that means that jars like this would have to be `org.locationtech.geomesa.<THING>.spark` right?

Reply:
the scheme i had settled on was to match module homing on file system...  base spark stuff was `org.locationtech.geomesa.spark.<extension>` then datastore extensions were `org.locationtech.geomesa.<extention>.spark`  this was a little munged w/ geotools and converter extentions.  I think this is Java11+ safe

Reply:
On the Java 11 front, I think jars can add things further down in a hierarchy.  I'm 95% sure about that:).